go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/speakeasy-api/speakeasy
 
-go 1.22
-
-toolchain go1.22.4
+go 1.22.4
```

Comment:
Unless there's some new language feature of 1.22.4, the prior was correct 😄 e.g. Using 1.22 language features and using a toolchain of a specific patch.